source-alpaca/.snapshots/TestSpec

```diff
@@ -0,0 +1,128 @@
+{
+  "endpoint_spec_schema_json": {
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/estuary/connectors/source-alpaca/config",
+    "properties": {
+      "api_key_id": {
+        "type": "string",
+        "title": "Alpaca API Key ID",
+        "description": "Your Alpaca API key ID.",
+        "secret": true
+      },
+      "api_secret_key": {
+        "type": "string",
+        "title": "Alpaca API Secret Key",
+        "description": "Your Alpaca API Secret key.",
+        "secret": true
+      },
+      "feed": {
+        "type": "string",
+        "enum": [
+          "iex",
+          "sip"
+        ],
+        "title": "Feed",
+        "description": "The feed to pull market data from. May be overridden within the binding resource configuration.",
```

Comment:
It feels like the whole "specifying feed/symbol in the endpoint config and then also/optionally in each resource" thing is going to be a source of pain in the future, but honestly I don't have a recommendation for how it could be done better.

Reply:
Yeah I think this is mostly vestigial from the way I had set up the configuration originally, with much more of the config at the resource-level rather than the endpoint-level. Upon reflection, I think it is best to remove these optional overrides entirely, as I don't think they will be used and they complicate things quite a bit.